packages/api/src/external/fhir/document/index.ts

```diff
@@ -387,14 +386,18 @@ function convertCWAdressToFHIR(address: Contained["address"] | undefined): Addre
  * @param genders CW gender code.
  * @returns FHIR-compliant gender string; otherwise undefined if a valid gender cannot be determined.
  */
-function convertCWGenderToFHIR(
-  genders: Gender[] | null | undefined
-): "male" | "female" | "other" | "unknown" | undefined {
-  if (genders) {
-    if (genders[0].code === "M") return "male";
-    else if (genders[0].code === "F") return "female";
-    else if (genders[0].code === "UN") return "other";
-    else if (genders[0].code === "UNK") return "unknown";
+function convertCWGenderToFHIR(genders: Gender[] | null | undefined): PatientFHIR["gender"] {
+  if (genders && genders[0]) {
```

Comment:
It was error'ing to compile/VSCode notifs, so I added the `tsconfig.json` to avoid this coding error. While there, also fixed the values being compared to `code` (they were not being checked in compile time, not they are`) and ended up moving to switch to simplify.